### PR TITLE
Get-missing-fonts-for-pdf-conversion

### PR DESCRIPTION
**Pull request by DharanyaSakthivel-SF4210**
Get-missing-fonts-for-pdf-conversion

### CODE REVIEW

File: Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                        // Save the PDF document to output stream
+                        using (FileStream outputStream = new FileStream(Path.GetFullPath(@"../../../Data/Result.pdf"), FileMode.OpenOrCreate, FileAccess.ReadWrite))
+                        {
+                            pdfDocument.Save(outputStream);
+                        }
+                    }
+                }
+            }
+
+            // Check and output the fonts that are not installed
```
Comment (MohanaselvamJothi) [inline Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs:38]: // Print the fonts that are not available in machine, but used in Word document.

File: Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            if (fonts.Count > 0)
+            {
+                Console.WriteLine("Fonts not installed in environment:");
+                foreach (string font in fonts)
+                    Console.WriteLine(font);
+            }
+            else
+            {
+                Console.WriteLine("Document used fonts are all installed.");
```
Comment (MohanaselvamJothi) [inline Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs:47]: "Fonts used in Word document are available in environment"

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs (hunk 1 above)
Comment (DharanyaSakthivel-SF4210) [reply Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs]: Modified

File: Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs (hunk 2 above)
Comment (DharanyaSakthivel-SF4210) [reply Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs]: Modified

File: Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                            pdfDocument.Save(outputStream);
+                        }
+                    }
+                }
+            }
+
+            // Check and output the fonts that are not installed
+            if (fonts.Count > 0)
+            {
+                Console.WriteLine("Fonts not installed in environment:");
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-to-PDF-Conversion/Get-missing-fonts-for-pdf-conversion/.NET/Get-missing-fonts-for-pdf-conversion/Program.cs]: Modified

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**